app/views/workflows/show.json.jbuilder

```diff
@@ -10,7 +10,9 @@ json.folder_contents (@workflow.folder_contents) do |content|
   json.set! 'type', Pathname(content).file? ? 'file' : 'dir'
   json.set! 'fsurl', Filesystem.new.fs(content)
   json.set! 'fs_base', Filesystem.new.fs(File.dirname(content))
-  json.set! 'terminal_base', OodAppkit.shell.url(path: File.dirname(content)).to_s
+  json.set! 'terminal_base', OODClusters[@workflow.batch_host] && OODClusters[@workflow.batch_host].login_allow? ?
+      OodAppkit.shell.url(path: File.dirname(content), host: @workflow.batch_host).to_s :
```

Comment:
Right now it isn't guaranteed that `login.host` will exist. Also I am fine with:

```
.../pun/sys/shell/ssh/owens.osc.edu/users/...
```

as we can't guarantee that another HPC center will have host aliases. If the admin wants to use `oakley` instead of `oakley.osc.edu`, then he should specify it in the cluster config as:

```yaml
...
login:
  host: "oakley"
```